src/app/_components/Navigation/DesktopNavigation/DesktopNavigation.tsx

```diff
@@ -44,10 +38,7 @@ export function DesktopNavigation() {
   return (
     <ul
       aria-label="Navigation items"
-      className={clsx(
-        'relative z-10 hidden lg:flex lg:items-center lg:gap-0.5',
-        TOUCH_TARGET_MAIN_NAV_ITEM.offsetWidth,
-      )}
+      className="relative z-10 hidden lg:flex lg:items-center lg:gap-0.5"
```

Comment:
I removed the offset as it had no effect on the element

Reply:
ah right! you think we could include it directly in the class? or prefer to be in the TOUCH_TARGET const?